gulpfile.js

```diff
@@ -14,6 +14,57 @@ require('laravel-elixir-vue-2');
  */
 
 elixir(mix => {
-    mix.sass('app.scss')
-       .webpack('app.js');
+    mix
+        .styles(
+            [
+                'slick-carousel/slick/slick.css',
+                'slick-carousel/slick/slick-theme.css',
+                'eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
+            ],
+            'public/css/bower.css',
+            './resources/assets/bower_components/'
+        )
+        .styles(
+            [
+                '*.css',
+            ],
+            'public/css/all.css'
+        );
+    mix
+        .scripts(
+            [
+                'jquery/dist/jquery.min.js',
+                'slick-carousel/slick/slick.min.js',
+                'angular-slick-carousel/dist/angular-slick.min.js',
+                'moment/min/moment.min.js',
+                'eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js',
+            ],
+            'public/js/bower.js',
+            './resources/assets/bower_components/'
+        )
+        .scripts(
+            [
+                'angular/common/filters.js',
+                'angular/common/directives/directives.js',
+                'angular/common/services/user-service.js',
+                'angular/common/services/fast-service.js',
+                'angular/common/services/news-service.js',
+                'angular/account/register.js',
+                'angular/soon/coming-soon.js',
+                'angular/about/about.js',
+                'angular/home/welcome.js',
+                'angular/home/new-fast.js',
+                'angular/nav/nav.js',
+                'angular/nav/sidebar.js',
+                'angular/app.js',
```

Comment:
If we can do '**/*.js' that would be great, cause then we don't need to continually add to this file.  Looks like there's a gulp.src plugin that can probably handle it.  I might be able to look into it at some point.